docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -12,10 +12,68 @@
       After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimizes the $|C^{-}|$.
       These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at the IP.
 
+## Rigid Waist Shift & LSA Knobs
+
+Some information about the rigid waist shift can be found in [this gallery page][rws_gallery]{target=_blank} from Felix's package's documentation.
+Some details about the creation of these knobs and their addition in LSA can be found in this [LHC OMC logbook entry](https://be-op-logbook.web.cern.ch/elogbook-server/GET/showEventInLogbook/3545713){target=_blank} and the ones that follow.
+
+### Knob Setting Convention
+
+Since the waist can be shifted in two directions - left and right of the IP - two knobs were generated per IP, for a unit setting of $1$ and -$1$.
```

Comment:
I do not understand this sentence.
The waist can be shifted to the left and the right, so from -1 (I assume left?) to 0 (IP) to 1 (right?), right? Why do you need two knobs?

Reply:
This is why I needed external eyes! It is a mistake from me: there is a single knob for the triplets per IP and two knobs for the rematching quadrupoles per IP. I will make corrections

Reply:
ah.. the other way around. ^^

Reply:
Adressed in [14abe46](https://github.com/pylhc/pylhc.github.io/pull/74/commits/14abe462aac917a757b1b6b3d51852e686029608)